000-debug/logger.php

```diff
@@ -94,8 +96,8 @@ function l( $stuff = null, ...$rest ) {
 			if ( array( 'default output handler' ) == $obs ) {
 				break;
 			}
-			// phpcs:ignore PHPCompatibility.FunctionUse.ArgumentFunctionsReportCurrentValue.NeedsInspection
-			$backtrace = debug_backtrace( false );
+			// phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_debug_backtrace
+			$backtrace = debug_backtrace( DEBUG_BACKTRACE_IGNORE_ARGS );
```

Comment:
Here and elsewhere: `debug_backtrace()` has been accepting `int` as its first parameter ([reference](https://www.php.net/debug_backtrace)) since at least PHP 7.3.

Judging by the code, the intention was to skip the arguments in the backtrace, that's why I replaced `false` with `DEBUG_BACKTRACE_IGNORE_ARGS`.